packages/integration-tests/projects/connect-popup/index.js

```diff
@@ -0,0 +1,370 @@
+                            log(f.method, v.name, "user interaction on host");
+                            await popup.click("button.confirm");
+                        } else if (confirm === 'device') {
+                            log(f.method, v.name, "user interaction on device");
+                            await controller.send({ type: 'emulator-press-yes' });
+                        } else {
+                            await popup.click(confirm);
+                        }
+                        confirmIndex++;
+                    }
+                    viewIndex++;
+                    log(f.method, v.name, "view finished");
+                }
+
+                log(f.method, "all views finished");
+
+                await explorer.waitForTimeout(2000);
+
+                // screenshot response
+                log(f.method, "screenshotting response");
+
+                await explorer.screenshot({ path: `${screenshotsPath}/4-response.png`, fullPage: true });
+                await explorer.waitForTimeout(2000);
+                log(f.method, "method finished");
+
+            }
+        }
+
+        log("closing browser");
+
+        await browser.close();
+
+        buildOverview();
+
+        process.exit(0);
+    } catch (err) {
+        console.log('err', err);
+        process.exit(1);
+    }
+
+})();
+
+const buildOverview = () => {
+    if (fs.existsSync('connect-popup-overview.html')) {
+        fs.rmSync('connect-popup-overview.html');
+    }
+
+    let html = '';
+    const methods = fs.readdirSync(SCREENSHOTS_DIR);
+    methods.forEach(method => {
+        const methodPath = path.join(SCREENSHOTS_DIR, method);
+        const screenshots = fs.readdirSync(methodPath);
+        html+= `<div><h1>${method}</h1></div>`;
+        screenshots.forEach(screenshot => {
+            const screenshotPath = `${CI_JOB_URL}/${methodPath}/${screenshot}`;
+            html += `
+                <div>
+                <img src="${screenshotPath.replace('/packages/connect-explorer', '/artifacts/raw/packages/connect-explorer')}" />
+                </div>
+            `;
+        })
+    });
+
+    // https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1843753200 /artifacts/raw /packages/connect-explorer/screenshots/composeTransaction/1-request.png
+    // https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1843753200                /packages/connect-explorer/screenshots/composeTransaction/1-request.png
+    
+    fs.appendFileSync('connect-popup-overview.html', `
+        <html>
+            <head>
+                <title>Connect popup</title>
+            </head>
+            <body>
+            ${html}
+            </body>
+        </html>
+    `);
+
+    // src={`${data.jobUrl}${screenshot.path.replace(
+    //     "trezor-suite",
+    //     "artifacts/raw"
+    //   )}`}
+
+}
+
+// todo: special cases
+// no bridge
+// outdated bridge
+// firmware update required
+// firmware outdated
```

Comment:
Are those comments all `todos`?
I am thinking if creating github issues for missing test cases will be more helpful that writing it here, otherwise it might be forgotten.

Reply:
I think this is not so important to create github issues. I am going to finish these soon I hope.